tests/__init__.py

```diff
@@ -335,7 +335,7 @@ def test_addons_env(self):
                 ("test", "-e", "auto/addons/crm"),
                 ("test", "-d", "auto/addons/crm/migrations"),
             )
-        for sub_env in matrix(odoo_skip={"11.0", "12.0", "13.0", "17.0"}):
+        for sub_env in matrix(odoo_skip={"11.0", "12.0", "13.0", "18.0"}):
```

Comment:
Why not testing all the possible versions (mostly recent ones - we can remove older ones)?